src/index.css

```diff
@@ -67,6 +83,36 @@ div:focus {
   outline: 0;
 }
 
+.list-block-height-container img, #footer-social-icons a, .carousel-button-container img, #desktop-coz-logo img, #mobile-coz-logo img, #chevron-right-transfer, #burger-menu-icon, .expanding-panel-container .explore-button img {
+  filter: invert(100%);
+}
+
+.dark-mode .list-block-height-container img, .dark-mode #footer-social-icons a, .dark-mode .carousel-button-container img, .dark-mode #desktop-coz-logo img, .dark-mode #mobile-coz-logo img, .dark-mode #chevron-right-transfer, .dark-mode #burger-menu-icon, .dark-mode .expanding-panel-container .explore-button img {
+  filter: invert(0%);
+}
+
+#neo-3-logo::after {
+  background: url("images/neo_color_dark.svg") no-repeat;
+  content: "";
+  width: 10em;
+  display: block;
+  height: 4em;
+}
+
+.dark-mode #neo-3-logo::after {
+  background: url("images/neo_color_light.svg") no-repeat;
+  content: "";
+  width: 10em;
+  display: block;
+  height: 4em;
+}
```

Comment:
It might be easier to just built a `<ThemedLogo />` component that follows similar logic to `<ThemeToggle />`so that we dont have to handle this in CSS not sure if that would make things easier in addressing the current issue I pointed out below

Reply:
Yes I kinda did a CSS hack instead .. will try and make something better tomorrow like you suggested.